src/main/java/com/minecolonies/core/client/gui/modules/FarmFieldsModuleWindow.java

```diff
@@ -171,6 +176,7 @@ public void updateElement(final int index, @NotNull final Pane rowPane)
                 if (field instanceof FarmField farmField && !farmField.getSeed().isEmpty())
                 {
                     rowPane.findPaneOfTypeByID(TAG_ICON, ItemIcon.class).setItem(farmField.getSeed());
+                    rowPane.findPaneOfTypeByID(TAG_STAGE, Text.class).setText(Component.literal(farmField.getFieldStage().toString()));
```

Comment:
I'd recommend an actual translation key for this, over just dumping the field state enum directly in view (we've been trying to move away for that with other things)